licenses.d2iq.yaml

```diff
@@ -96,11 +96,11 @@ resources:
       - license_path: LICENSE
         ref: v1.19.2-d2iq.1
         url: https://github.com/mesosphere/gitea
-  - container_image: docker.io/grafana/grafana:10.3.3
+  - container_image: ghcr.io/mesosphere/dkp-container-images/docker.io/grafana/grafana:10.3.3-d2iq.0
     sources:
       - license_path: LICENSE
         notice_path: NOTICE.md
-        ref: v${image_tag}
```

Comment:
Can you use envsubst and not hardcode the ref tag?

Reply:
updated !!